### ISSUE TITLE
what to put in the README?

### ISSUE THREAD

**Issue by petrelharp** [popgensims/demographicModels#1]
Very nice example! We should think of what should go in READMEs for examples in general?

Ideas:

1. Model description
2. Citation(s)
3. How to run it
4. How long it takes to run, more or less

anything else?  @popgensims/all?

**Comment by petrelharp** [comment]
@andrewkern suggests that we have subdirectories for different modeling frameworks (eg one for msprime, one for SLiM). I think this is a great idea.

**Comment by andrewkern** [comment]
yes it would be good to set up conventions early here. perhaps we can stub out a simple template for each model readme?

**Comment by tsackton** [comment]
Katie Lotterhos and I spent some time discussing some of these issues this afternoon (working on an email to send to the listserv to hopefully prompt some discussion). It seems like it makes sense to start with having one directory for each simulation script, and a README that includes:

1) description of files in the directory (possibly just the simulation script, but if there is a simulation recipe for varying a parameter like Ne this could include additional pre- or post-processing scripts, and if we decide to store output this would include description of output as well). 

2) Short verbal description of the model (e g. Human demographic model based on Tennison et al 2012)

3) Requirements for running the model (Python / SLIM / msprime / etc versions, at least)

4) Citation if the model is based on published work.

Thoughts?

**Comment by petrelharp** [comment]
Here's a proposal; feel free to modify.  Example file structure
```
README
plot_results.py
slim/
---> DESCRIPTION
---> (script).slim
---> postprocess.py
msprime/
---> DESCRIPTION
---> (script).py
---> do_other_stuff.sh
---> more_stuff.py
```

In the first `README`:
```
# Description:
Short description of the model, in sentence form.

# Contents:

- plot_results.py : what does this do
- slim/ : the model, in SLiM
- msprime/: the model, in msprime

# Citations:
Citation(s).
```

and then in the `DESCRIPTION` files, one per subdirectory:
```
# Quickstart

Basic command for how to run the model.

# Requirements

Software requirements, installation instructions.
Some idea of how long this takes to run.

# Parameters

Description of adjustable parameters.

# Contents:
List of files in the directory, and what they do

- script.slim : the SLiM script
- postprocess.py : what does this do
```

... how's that?

**Comment by tsackton** [comment]
That looks pretty reasonable to me. I am in meetings all day today but hopefully will have some time tomorrow to play around, I think starting to build up the Tennesen model as an example is probably a good place to start.
